### PR TITLE
some changes to support fine-tuning on Intel GPU

### PR DESCRIPTION
**Pull request by harborn**

### CODE REVIEW

File: common/trainer/default_trainer.py [hunk 1]
```diff
@@ -153,29 +151,37 @@ def prepare(self, model, tokenizer, dataset, optimizer, accelerator):
     def train(self):
         num_train_epochs = self.config.get("num_train_epochs", 1)
         checkpoint = self.config.get("checkpoint")
-        log_step = self.config.get("log_step", 1)
+        logging_steps = self.config.get("logging_steps", 1)
         max_train_step = self.config.get("max_train_step")
         max_eval_step = self.config.get("max_eval_step")
+        gradient_accumulation_steps = self.accelerator.gradient_accumulation_steps
+        output = self.config.get("output", "./output")
+        writer = torch.utils.tensorboard.SummaryWriter(output)
```
Comment (KepingYan) [inline common/trainer/default_trainer.py:159]: I don’t think the `writer` is needed, maybe ray.train.report is enough. After running `tensorboard --logdir .../ray_results/TorchTrainer_2024* --bind_all` we can see the parameters set in `report()`. @carsonwang please help confirm this.

**Review by jiafuzha — APPROVED** [review]
Approve it first for later dependent PR

**Comment by xwu99** [comment]
CI failed when merged. could you check? @harborn

**Comment by jiafuzha** [comment]
> CI failed when merged. could you check? @harborn

[10:04 AM] Zhang, Jiafu
应该是login 失效了，我重新login一下
[10:04 AM] Zhang, Jiafu
对，但是以前build失败都会抱出来
[10:07 AM] Zhang, Jiafu
现在应该好了，我在2.18重新login了一次
[10:08 AM] Zhang, Jiafu
你可以叫tianchen写一个relogin的脚步，这个docker login 没有参数让login永远不失效
[10:08 AM] Zhang, Jiafu
docker login https://amr-cache-registry.caas.intel.com/

**Comment by xwu99** [comment]
> > CI failed when merged. could you check? @harborn
> 
> [10:04 AM] Zhang, Jiafu 应该是login 失效了，我重新login一下 [10:04 AM] Zhang, Jiafu 对，但是以前build失败都会抱出来 [10:07 AM] Zhang, Jiafu 现在应该好了，我在2.18重新login了一次 [10:08 AM] Zhang, Jiafu 你可以叫tianchen写一个relogin的脚步，这个docker login 没有参数让login永远不失效 [10:08 AM] Zhang, Jiafu docker login https://amr-cache-registry.caas.intel.com/

the login was fixed. but it failed anyway. I would revert before it's fixed.

**Pull request merged**